server/newTransactionSubmitter.js

```diff
@@ -0,0 +1,48 @@
+require('isomorphic-fetch')
+
+async function request(url, method = 'get', body = null, headers = {}) {
```

Comment:
no need for this function - it is used only once, so I would prefer using fetch directly